x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportStartDatafeedAction.java

```diff
@@ -251,6 +252,21 @@ public void onFailure(Exception e) {
                         params.setJobId(datafeedConfig.getJobId());
                         params.setIndicesOptions(datafeedConfig.getIndicesOptions());
                         datafeedConfigHolder.set(datafeedConfig);
+                        if (datafeedConfig.hasCompositeAgg(xContentRegistry)) {
+                            if (state.nodes()
+                                .mastersFirstStream()
+                                .filter(MachineLearning::isMlNode)
+                                .map(DiscoveryNode::getVersion)
+                                .anyMatch(COMPOSITE_AGG_SUPPORT::after)) {
```

Comment:
Its the same as `(discoveryNodeVersion -> COMPOSITE_AGG_SUPPORT.after(discoveryNodeVersion))`. So, is `COMPOSITE_AGG_SUPPORT` after the `discoveryNodeVersion`.

If it were to use `before` I think it would be `Version::before` to execute on the instance of the `Version` class in the stream.